source/style/markdown.css

```diff
@@ -0,0 +1,7 @@
+code.refined_twitter_markdown {
+  font-family: Inconsolata, Monaco, Consolas, 'Courier New', Courier, monospace;
+  background-color: rgba(27,31,35,0.05);
+  padding: 0.1em 0.4em;
+  margin: 0;
+  font-size: 90%;
+}
```

Comment:
I don't see the benefit of putting this in a separate file. Can just be in `content.css`.